src_c/math.c

```diff
@@ -1633,18 +1633,49 @@ vector_reflect_ip(pgVector *self, PyObject *normal)
 static double
 _vector_distance_helper(pgVector *self, PyObject *other)
 {
-    Py_ssize_t i;
-    double distance_squared, tmp;
+    Py_ssize_t i, dim = self->dim;
+    double distance_squared = 0;
+
+    if (pgVector_Check(other)) {
```

Comment:
A comment that explains that this is a fastpath for performance reasons would help here, so that people looking at this code a few years down the line don't think that this is redundant and remove it

Reply:
ups missed this one